man/systemd-boot.xml

```diff
@@ -461,6 +461,23 @@
         <xi:include href="version-info.xml" xpointer="v240"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>LoaderEntryLastBooted</varname></term>
+
+        <listitem><para>The identifier of the boot loader entry last attempted. Set and read by the boot loader.
+        The boot loader modifies <varname>LoaderEntryLastBooted</varname> on every boot, unless
+        <varname>LoaderEntryOneShot</varname> is set.</para>
+
+        <para>The boot loader reads the variable, which takes higher priority than
+        <varname>LoaderEntryDefault</varname>. The variable is ignored when <varname>LoaderEntryOneShot</varname>
+        is set.</para>
```

Comment:
note this is only used if "default" is set to "@saved" in loader.conf. This deserves notice.